### ISSUE TITLE
Completion selects based on insertion string

### ISSUE THREAD

**Issue by svenefftinge** [Microsoft/monaco-editor#232]
version: 0.7.3

How to reproduce: Go to playground completion example and use the completions below.
In the example editor type in 'a' and see how 'abd' proposal gets auto-selected. It should select 'abc' instead.

```
function createDependencyProposals() {
    // returning a static list of proposals, not even looking at the prefix (filtering is done by the Monaco editor),
    // here you could do a server side lookup
    return [
        {
            label: 'abc',
            kind: monaco.languages.CompletionItemKind.Function,
            insertText: '"abc": "*"'
        },
        {
            label: 'abd',
            kind: monaco.languages.CompletionItemKind.Function,
            insertText: 'abd',
        }
    ];
}
```

**Comment by svenefftinge** [comment]
Could be caused by this commit : https://github.com/Microsoft/vscode/commit/11d23f1562700c4f072f10256b4b68e0db8c5c0a

**Comment by svenefftinge** [comment]
Seems to be this issue https://github.com/Microsoft/vscode/issues/11423

**Comment by svenefftinge** [comment]
Do you have an estimate when the fix will be in a monaco release?

**Comment by aarinsmith** [comment]
@svenefftinge @joaomoreno mentioned in my issue here https://github.com/Microsoft/monaco-editor/issues/219 that this has been resolved and should be included in next release.

**Comment by alexandrudima** [comment]
It looks like the fix for #11423 landed in master 5 days ago, so it will be contained in the next `monaco-editor` release, due at the end of this month / early beginning of next month. We ship it every month and won't create a new build unless we find some critical regression.

**Comment by joaomoreno** [comment]
Closing as duplicate.
